css/includes/components/form/_form-editor.scss

```diff
@@ -370,6 +370,12 @@
     border-bottom-right-radius: var(--tblr-border-radius) !important;
 }
 
+input.value-selector {
```

Comment:
not ok with this one, as noted in [the initial pr ](https://github.com/glpi-project/glpi/pull/18874), this is a minor issue found in another place, but I considered it not blocking for the merge.

But for the fix, we must make it globally, and not not in forms.

Reply:
Noted, let's look for a global fix after this one is merged.